### PR TITLE
CXSPA-2012: fix unit-level orders styling issues

### PR DESCRIPTION
**Pull request by MarcinGanczorz**
https://jira.tools.sap/browse/CXSPA-2012
- replace static colors with cx- variables
- rename css classes, add cx- prefix
- center pagination on mobile view
- change structure of mobile filter view to use flexbox positioning instead of float
- adjust font and icon sizes for mobile filter view

### CODE REVIEW

File: feature-libs/organization/unit-order/components/unit-level-order-history/filter/unit-level-order-history-filter.component.ts [hunk 1]
```diff
@@ -117,6 +117,7 @@ export class UnitLevelOrderHistoryFilterComponent {
   }
 
   closeFilterNav(): void {
+    console.log('closeFilterNav');
```
Comment (I539321) [inline feature-libs/organization/unit-order/components/unit-level-order-history/filter/unit-level-order-history-filter.component.ts]: Drop this `console.log`, along with the one in the `launchSubNav` method.

**Review by I539321 — CHANGES_REQUESTED** [review]

**Comment by pavel-ermolin** [comment]
looks good

File: feature-libs/organization/unit-order/components/unit-level-order-history/filter/unit-level-order-history-filter.component.ts (hunk 1 above)
Comment (I539321) [reply feature-libs/organization/unit-order/components/unit-level-order-history/filter/unit-level-order-history-filter.component.ts]: Resolving this conversation

**Review by I539321 — APPROVED** [review]

**Comment by I539321** [comment]
Looks okay based on a general overview

**Pull request merged**